spotify_connect_scrobbler/lastfm.py

```diff
@@ -37,55 +33,17 @@ def sign(self, parameters):
         md5.update(string.encode('utf-8'))
         return md5.hexdigest()
 
-    def request_authorization(self, redirect_uri):
```

Comment:
I'd keep this as part of the library. Especially the authorization for Last.fm is a little weird. IMHO this should be part of the library since it's also part of the API.